includes/discovery/sensors/state/fortigate.inc.php

```diff
@@ -19,8 +19,8 @@
  *
  * @link       https://www.librenms.org
  *
- * @copyright  2020 Net Entertainment AB
- * @author     Patrik Jonsson <patrik.jonsson@gmail.com>
+ * @copyright  2020 Net Entertainment AB, 2024 CTNET BV
```

Comment:
It actually should be additional @author and @copyright lines :)

Reply:
Fixed!